tests/app/v1_basic_test.py

```diff
@@ -1,10 +1,23 @@
 """
 Basic Test for V1 of API
 """
+import psycopg2
 import pytest
+import sqlalchemy
+import testing.postgresql
+
+import app
 
 from flask import json, jsonify
 
+@pytest.fixture
```

Comment:
I wouldn't delete conftest.py, and afaict keeping `create_app` with a config is idiomatic. I think we want to do something like the PyTest section of https://spotofdata.com/flask-testing/

Reply:
Oh, yea, that looks right for initializing that through the config, namely:

```
from testing.postgresql import Postgresql

_app = create_app(TestConfig)  # TestConfig is defined further down
with Postgresql() as postgresql:
    _app.config['SQLALCHEMY_DATABASE_URI'] = postgresql.url()
```


I'm still super wigged out by the stringly-typed magic config going on in there. I guess https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY serves as some documentation, but things like the database config are also separate magic coming from...somewhere else?